v2/pkg/output/file_output_writer.go

```diff
@@ -12,12 +12,20 @@ type fileWriter struct {
 }
 
 // NewFileOutputWriter creates a new buffered writer for a file
-func newFileOutputWriter(file string) (*fileWriter, error) {
-	output, err := os.Create(file)
-	if err != nil {
-		return nil, err
+func newFileOutputWriter(file string, resume string) (*fileWriter, error) {
```

Comment:
What if we rewrite it this way?
```go
// NewFileOutputWriter creates a new buffered writer for a file
func newFileOutputWriter(file, resume string) (*fileWriter, error) {
	var flags int
	if resume != "" {
		flags = os.O_APPEND | os.O_CREATE | os.O_WRONLY
	} else {
		flags = os.O_CREATE
	}
	output, err := os.OpenFile(file, flags, 0644)
	if err != nil {
		return nil, err
	}
	return &fileWriter{file: output}, nil
}
```

Reply:
ok you're right, this is better, I implement it